engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js

```diff
@@ -24,6 +24,8 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsController
         $scope.getHostStatusIcon = ContentHostsHelper.getHostStatusIcon;
         $scope.getHostPurposeStatusIcon = ContentHostsHelper.getHostPurposeStatusIcon;
 
+        $scope.organization = Organization.get({id: CurrentOrganization});
+
```

Comment:
@johnpmitsch turns out the surrounding `if` wasn't needed.

the execution of the code is like this:

- load content host details page
- controller JS is executed and $scope.organization becomes an angular resource - whether or not the server has returned is another story
- open dropdown for any of the syspurpose fields, loading is deferred until we have an org (and ideally we will by then!)

I verified that we are still making only 1 request to /organizations/:id across the different dropdowns

Reply:
Thanks for testing it out! Maybe we are starting to actually use `$resource` correctly? :thinking:

Reply:
better late than never!